library/proc_macro/src/lib.rs

```diff
@@ -109,6 +109,34 @@ impl !Send for LexError {}
 #[stable(feature = "proc_macro_lib", since = "1.15.0")]
 impl !Sync for LexError {}
 
+/// Error returned from `TokenStream::expand_expr`.
+#[unstable(feature = "proc_macro_expand", issue = "none")]
+#[non_exhaustive]
+#[derive(Debug)]
+pub struct ExpandError;
+
+impl ExpandError {
+    fn new() -> Self {
+        ExpandError
+    }
```

Comment:
This function is trivial and private, so it's shouldn't be necessary.

Reply:
I did it for symmetry with `LexError`, but can remove if you want.

Reply:
Yes, `LexError::new` can be removed as well.